pkg/syncer/syncer.go

```diff
@@ -73,15 +73,58 @@ func (sc *SyncerConfig) ID() string {
 func StartSyncer(ctx context.Context, cfg *SyncerConfig, numSyncerThreads int, importPollInterval time.Duration) error {
 	klog.Infof("Starting syncer for logical-cluster: %s, workload-cluster: %s", cfg.KCPClusterName, cfg.WorkloadClusterName)
 
+	kcpClusterClient, err := kcpclient.NewClusterForConfig(rest.AddUserAgent(rest.CopyConfig(cfg.UpstreamConfig), "kcp#syncer/v0.0.0"))
+	if err != nil {
+		return err
+	}
+
+	// For now we retrieve the syncerVirtualWorkpaceURL at start, since we temporarily stick to a single URL (sharding not supported).
```

Comment:
Please make sure this comment has `TODO` in there somewhere - thanks!

Reply:
Fixed